app/View/Components/Notification/EstimateRequested.php

```diff
@@ -13,16 +13,18 @@ class EstimateRequested extends Notification
 {
     public Project $project;
 
-    public Organization|RegulatedOrganization $projectable;
+    public mixed $projectable;
 
     public function __construct(DatabaseNotification $notification)
     {
         $this->project = Project::find($notification->data['project_id']);
         $this->projectable = $this->project->projectable;
+        /** @var Organization|RegulatedOrganization */
+        $projectable = $this->projectable;
```

Comment:
@greatislander can line 21 and 22 be combined?
```suggestion
        /** @var Organization|RegulatedOrganization */
        $projectable = $this->project->projectable;
```